index.html

```diff
@@ -45,5 +45,194 @@ <h1 class="title">I'm Mariane <br>Glad to see you!</h1>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>
+        </div>
+
+        <div class="works">
+            <img src="images/Snapshoot Portfolio (2).svg" alt="" class="tonic">
+            <img src="images/Snapshoot Portfolio3.svg" alt="" class="tonic-1">
+            <div>
+                <h1 class="header-tonic">Tonic</h1>
+
+                <ul class="description">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">Back End Dev</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">2015</li>
+                </ul>
+
+                <p class="more-info">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <ul class="languages">
+                    <li><button class="lang" id="lan-1">html</button></li>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>
+        </div>
+
+        <div class="works works2">
+            <img src="images/Snapshoot Portfolio (3).svg" alt="" class="tonic">
+            <img src="images/Snapshoot Portfolio4.svg" alt="" class="tonic-1">
+            <div>
+                <h1 class="stories">Multi-Post Stories</h1>
+
+                <ul class="description">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">Back End Dev</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">2015</li>
+                </ul>
+
+                <p class="more-info">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <ul class="languages">
+                    <li><button class="lang" id="lan-1">html</button></li>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-myself" id="about-myself">
+        <div class="myself">
+            <div>
+                <h1 class="about">About Myself</h1>
```

Comment:
- [ ] In the [figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=zjF2gQCiVeUiBNY5-0) the `About Myself` topic should be in one line, and as we are trying to aim for a pixel perfect design I would suggest you adjust your heading size to fit in one line as well


| Figma design | Your design |
|--------------|--------------|
|<img width="285" alt="Screenshot 2023-04-29 at 5 29 19 PM" src="https://user-images.githubusercontent.com/44798044/235302798-16e3ce48-6c57-4b5b-af3d-5116925e5c33.png">|<img width="309" alt="Screenshot 2023-04-29 at 5 29 36 PM" src="https://user-images.githubusercontent.com/44798044/235302803-6f7f87fb-a75f-4bb4-a981-9d2e548d7fe7.png">|

Reply:
Fixed it into one line